src/analyzer.rs

```diff
@@ -225,6 +228,7 @@ fn analyze_enum_properties(
         level,
         docs: schema.description.clone(),
         is_enum: true,
+        supports_derive_default: OnceCell::new(),
```

Comment:
can avoid importing and doing this with
```suggestion
        ..Container::default()
```